### PR TITLE
Scraper admin page - improved reloading behavior

### PR DESCRIPTION
**Pull request by martinraison**
@raymondng @eishay 
Refreshing the list of scraping requests does not reload the whole page, and the auto-refresh can be enabled/disabled.

### CODE REVIEW

**Comment by raymondng** [comment]
Nice!

**Comment by ymatsuda** [comment]
Thanks!

File: kifi-backend/modules/shoebox/app/views/admin/searchScraper.scala.html [hunk 1]
```diff
@@ -44,6 +27,23 @@
     </form>
 
     <script>
-            setInterval(function(){location.reload();}, 5000);
+        var refresh = function(){
+            if ($("#autoreload").is(":checked")) {
+                $.ajax({
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/views/admin/searchScraper.scala.html:32]: Oooo... I see frontend potential here! Careful or you may get drafted. `;)`

**Comment by martinraison** [comment]
All credit goes to StackOverflow :)

**Pull request merged**